### PR TITLE
feat: Added Signup Page

### PR DESCRIPTION
**Pull request by Ankur9669**
image

### CODE REVIEW

File: signup.css [hunk 1]
```diff
@@ -0,0 +1,131 @@
+.signup{
+    display: grid;
+    grid-template-columns: 50% 50%;
+    background-color: var(--light-blue);
+    border-radius: 8px;
+    width: 90%;
+    max-width: 93.7rem;
+    margin: 0rem auto;
```
Comment (sri-rishi) [inline signup.css:8]: Just a suggestion you can use 0 except for 0rem

**Review by sri-rishi — CHANGES_REQUESTED** [review]
Otherwise looks  good to merge

**Review by Naman-Saxena1 — APPROVED** [review]
Can't find any major issues, looks good to merge 👍

**Comment by eakansh125** [comment]
Syntactically, the code looks good to me to be merged.
Just some suggestions:
1. Instead of writing 0rem/0px, we can write just 0.
2. While focusing on the Email and password textbox, a very slight shift taking place can be seen. I tacked this situation by keeping the border size same in the textbox class as well as in the focus class.

**Comment by Rishabh-078** [comment]
Some suggestions:
a. Classnames appear to be lengthy
b. rem and px can be dropped with "0"

Rest code and UI are looking good.

**Comment by swstica** [comment]
In my opinion, font size in the footer can be decreased. Rest all looks good to me:)

File: signup.css [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    justify-content: center;
+    margin: 1.8rem 0rem;
+}
+.form-input-box input{
+    width: 90%;
+    padding-left: 1rem;
+    height: 3.12rem;
+    box-shadow: rgba(0, 0, 0, 0.1) 0px 4px 12px;
+    border-radius: 3px;
+    border: 1px solid #BCBCBC;
```
Comment (Priya-Patel31) [inline signup.css]: use variables for color

File: signup.css [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    font-size: 1.5rem;
+}
+.checkbox-container{
+    width: 90%;
+    display: flex;
+    justify-content: space-between;
+    margin: 0rem auto 1.8rem auto;
+}
+.forgot-password-btn{
+    color: black;
```
Comment (Priya-Patel31) [inline signup.css]: use variables for color

File: signup.css [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+    background-color: white;
+    max-width: 31.2rem;
+    width: 100%;
+    border-radius: 6px;
+    padding: 1rem 0.5rem;
+    box-shadow: rgba(0, 0, 0, 0.16) 0px 3px 6px, rgba(0, 0, 0, 0.23) 0px 3px 6px;
+}
+.authentication-form-heading{
+    text-align: center;
+    margin: 0rem 1rem 0.3rem 1rem;
```
Comment (Priya-Patel31) [inline signup.css:58]: 0 is just enough rather than using 0rem

File: signup.html [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+        <a class="button button-primary" href="login.html">Login</a>
+      </div>
+    </div>
+  </nav>
+  <!--  ******************************** Navbar End ********************************  -->
+
+  <!--  ******************************** SignUp Container Start ********************************  -->
+  <div class="spacer-3"></div>
+  <main>
+    <img src="images/blob.svg" class="blob blob1" />
```
Comment (Priya-Patel31) [inline signup.html]: use alt attribute as it is a good practice !!

**Review by Priya-Patel31 — APPROVED** [review]
Everything seems good to me and code quality is very good !!!

File: signup.css (hunk 1 above)
Comment (Ankur9669) [reply signup.css:8]: Ya i can use but i prefer this more😅

**Comment by Ankur9669** [comment]
> Some suggestions: a. Classnames appear to be lengthy b. rem and px can be dropped with "0"
> 
> Rest code and UI are looking good.

For the 1st point, i like to keep things consistent that is why i like to use 0rem instead of 0
For the 2nd point, That is a very nice catch i will correct it right away

**Comment by Ankur9669** [comment]
> In my opinion, font size in the footer can be decreased. Rest all looks good to me:)

Ok sure i will look into that

File: signup.css (hunk 2 above)
Comment (Ankur9669) [reply signup.css]: Its just a border that is why i used this only

File: signup.css (hunk 3 above)
Comment (Ankur9669) [reply signup.css]: Sure

**Comment by Ankur9669** [comment]
> Some suggestions: a. Classnames appear to be lengthy b. rem and px can be dropped with "0"
> 
> Rest code and UI are looking good.

Ya the class names are lengthy but i cannot do much as i am following OOCSS concept😅

**Comment by Ankur9669** [comment]
> Syntactically, the code looks good to me to be merged. Just some suggestions:
> 
> 1. Instead of writing 0rem/0px, we can write just 0.
> 2. While focusing on the Email and password textbox, a very slight shift taking place can be seen. I tacked this situation by keeping the border size same in the textbox class as well as in the focus class.

Done

File: signup.html (hunk 5 above)
Comment (Ankur9669) [reply signup.html]: Done

File: signup.css (hunk 4 above)
Comment (Ankur9669) [reply signup.css:58]: Actually i like to keep things consistent that's why i use them

File: signup.css (hunk 3 above)
Comment (Ankur9669) [reply signup.css]: Done

File: signup.css (hunk 1 above)
Comment (sri-rishi) [reply signup.css:8]: looking good to merge

**Review by sri-rishi — APPROVED** [review]

**Pull request merged**